CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/docs/expressions.html

```diff
@@ -0,0 +1,225 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>The Constellation Expressions Framework</title>
+        <link rel="stylesheet" href="nbdocs://au.gov.asd.tac.constellation.preferences/au/gov/asd/tac/constellation/preferences/constellation.css" type="text/css">
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    </head>
+    <body>
+        <h1>The Constellation Expressions Framework</h1>
+        <p>
+            The expressions framework is a structure which can be used to query or assign the Vertex and Transaction attributes within a graph.
+            The framework will work on all attributes which <strong>do not</strong> include a restricted character within it's AttributeName. An expression can 
+            return <strong>0..*</strong> (none or many) matches.
```

Comment:
- remove strong tags
- replace "return 0..* (none or many)" with simply "returns zero or more"
- surround AttributeName in double-quotes